client/src/hooks/usePatchArticle.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { patchArticle } from '../api/articles';
+
+export default function usePatchArticle() {
+  const [article, setArticle] = React.useState(null);
+  const [errorArticle, setErrorArticle] = React.useState(false);
+  const [loadingArticle, setLoadingArticle] = React.useState(true);
+
+  async function doPatchArticle(
+    articleId,
+    articlenumber,
+    client,
+    bbd,
+    pzn,
+    ean
+  ) {
+    try {
+      const article = await patchArticle(
+        articleId,
+        articlenumber,
+        client,
+        bbd,
+        pzn,
+        ean
+      );
```

Comment:
You could save some lines of code if you change your functions to accept only one or two arguments:

```suggestion
  async function doPatchArticle(
    articleId,
    article
  ) {
    try {
      const article = await patchArticle(
        articleId,
        article
      );
```

or

```suggestion
  async function doPatchArticle(
    article
  ) {
    try {
      const article = await patchArticle(
        articleId,
      );
```